src/Microsoft.AspNetCore.Routing.Abstractions/Address.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public class Address
```

Comment:
How do you imagine that MVC's URL generation will work (take `Url.Action` as an example)